### DISCUSSION TITLE
I2V with start and end keyframes

### DISCUSSION THREAD

**Discussion by PointReact** [THUDM/CogVideo#447] [General]
I just want to say that this is a great project with very impressive results, equivalent or better than some of the commercial products out there but possible to run on a home machine.  

If you could specify a start and end frame for image-to-video (like is possible with tooncrafter), then this would be a truly next level tool.  Please consider this.

**Comment by h5bc7nyv6j-source** [comment]
<img width="1536" height="2244" alt="0547C3E5-7F1E-4027-9A60-16EDD7D106F9" src="https://github.com/user-attachments/assets/8abf9036-6fa3-4ca7-a481-bee9e502ca1c" />
